app/views/orga/seasons/index.html.slim

```diff
@@ -27,3 +27,5 @@ table.table.table-striped.table-bordered.table-condensed
 br
 
 = link_to 'New Season', [:new, :orga, :season], class: 'btn btn-default'
+
+= render partial: 'orga/seasons/switch' if Rails.env.development?
```

Comment:
The test env is exactly what I had in mind: checking for developement only would skip that portion during our tests, meaning you could literally place a `fail` statement in the template without noticing it running the specs.

Every test should run on a clean test DB. If we find the tests are randomly failing, we may have spotted an example leaking data and/or a problem with the database cleansing.